django_app/report_a_suspected_breach/templates/report_a_suspected_breach/base_form_step.html

```diff
@@ -4,13 +4,21 @@
 {% block extra_css %}
     {{ wizard.form.media.css }}
 {% endblock extra_css %}
+{% block title %}
+    {% if form_h1_header %}
+        <title>{{form_h1_header}} - Report a suspected breach of trade sanctions - GOV.UK</title>
+    {% else %}
+        <title>{% for field in wizard.form %}{% if forloop.first %}{{ field.label }}{% endif %}{% endfor %} - Report a suspected breach of trade sanctions - GOV.UK</title>
```

Comment:
the loop feels quite wasteful, I think an index would be better

```suggestion
        <title>{{ wizard.form.fields.0.label }} - Report a suspected breach of trade sanctions - GOV.UK</title>
```

There will also be cases where the first 'label' is actually a legend (e.g. radio buttons)- not sure if this can handle that

Reply:
I tried that previously but fields.0.label doesn't render anything, I think its because wizard.form.fields returns an ordered dict rather than a list. 
It only accesses the first field's label so will ignore the legend's label in that case - eg WhereWereTheGoodsSuppliedFrom page